starknet_py/contract.py

```diff
@@ -238,9 +238,13 @@ async def invoke(self, *args, **kwargs) -> InvocationResult:
         """
         return await self.prepare(*args, **kwargs).invoke()
 
-    @property
-    def selector(self):
-        return get_selector_from_name(self.name)
+    @staticmethod
```

Comment:
This is just a wrapper for `get_selector_from_name`. What do we gain by this change?

Reply:
We're requiring the user to pass the selector to the L1 message dataclass, and since it's hidden inside our dependency, user would have to import that from there. We can expose it this way to make it easier